cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -110,6 +111,7 @@ resources:
 - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/scc-privileged-opf-ci-pipelines-aicoe-ci
 - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/tekton-dashboard
 - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/vault-server-binding
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/vault
```

Comment:
can we create a vault bundle here: https://github.com/operate-first/apps/tree/master/cluster-scope/bundles

And add the namespace/rolebindings to it, then import the bundle instead of the 2 resources separately?